apps/dialtone-documentation/docs/.vuepress/baseComponents/TokenTable.vue

```diff
@@ -161,24 +164,36 @@ export default {
     },
 
     onEnterRow (name) {
-      console.log('ENTER', name);
       this.hoveredRow = name;
     },
 
-    onLeaveRow (name) {
-      console.log('LEAVE', name);
+    onLeaveRow () {
       this.hoveredRow = null;
     },
 
     showCopyButton (name) {
       return this.hoveredRow === name;
     },
+
+    onFocusOutRow () {
+      if (document.activeElement.tagName === 'TR') {
+        this.hoveredRow = null;
+      }
+    },
+
+    async onFocusInRow (name) {
+      if (this.hoveredRow === name) return;
+      this.hoveredRow = name;
+      // when focusing the next row we want to redirect the focus to
+      // the copy button
+      if (document.activeElement.tagName === 'TR') {
+        await this.$nextTick(); // wait for the copy button to render
+        const copyButtonRef = this.$refs[`copyButton${name}`];
+        if (Array.isArray(copyButtonRef)) {
+          copyButtonRef[0]?.$el?.querySelector('button')?.focus();
+        }
+      }
+    },
```

Comment:
Makes sense. @yorbi-dp if you agree I would go with that approach, this is how it would work:
![tokens-nav](https://github.com/dialpad/dialtone/assets/24460973/2afa616f-4314-4985-bc5e-e1a600c81b2f)

Reply:
I see your point; focusing on the row first, then the copy button does seem more natural, especially for accessibility purposes to ensure the screen reader reads the value. However, since the row doesn't have any associated action or navigation, unlike the contact-row which likely opens a chat, there might not be a pressing need to focus on it. Alternatively, adding an aria-label to the button, such as 'Copy <token name> to clipboard,' could be a good solution. What do you guys think?

Reply:
The button has an aria-label that is "copy token", the thing is that without the focus on the row you cannot really read the name, value and description. Unless there's a way of doing that with aria attributes, but I don't think it's posible doing it this way because since the button is not initially visible, I need to remove the focus from the row to the button once it's visible and that messes with the accessibility.
I would go with the focus in the row even if it doesn't lead to an action, because we want to prioritize accessibility.

Reply:
Oh, got it. Awesome, let's do the focus on the row then. Thank you! 🙌